tests/conftest.py

```diff
@@ -27,7 +27,7 @@ def is_excluded_frontend_file(path):
 
 def pytest_addoption(parser):
     parser.addoption("--disable_sql_simplifier", action="store_true", default=False)
-    parser.addoption("--local_testing_mode", action="store_true", default=False)
+    parser.addoption("--local_testing_mode", action="store_true", default=True)
```

Comment:
why is this change here?

Reply:
reverted, i was runing test locally with local testing mode